cmd/adapter/main.go

```diff
@@ -69,7 +71,10 @@ func (a *Adapter) makeProviderOrDie() provider.MetricsProvider {
 		os.Exit(1)
 	}
 
-	return kedaprovider.NewProvider(logger, handler, kubeclient, namespace)
+	metricsServer := prommetrics.NewPrometheusMetricsServer()
+	go func() { metricsServer.StartServer(":9022", "/metrics") }()
```

Comment:
Question - Should we make port + path configurable with this as default?

Reply:
Ok looks like the port is already configurable based on the docs?

> The KEDA Metrics Adapter exposes Prometheus metrics which can be scraped on port `9022` (this can be changed by setting the `metrics-port` argument for the Metrics Adapter) at `/metrics`.

So we have to update our Helm chart with this or does it already exist?

Reply:
> Question - Should we make port + path configurable with this as default?

I'll add in the path as a configurable

Reply:
> Ok looks like the port is already configurable based on the docs?
> 
> > The KEDA Metrics Adapter exposes Prometheus metrics which can be scraped on port `9022` (this can be changed by setting the `metrics-port` argument for the Metrics Adapter) at `/metrics`.
> 
> So we have to update our Helm chart with this or does it already exist?

The Helm Chart could be updated to specify these. I was planning on updating the Helm Chart to add the ability to either add the Prometheus scrape annotation or the Prometheus Operator Pod Monitor resource for the Metrics Adapter (this is quite common in other Helm Charts with Prometheus Metrics)